### DISCUSSION TITLE
1.5.2

### DISCUSSION THREAD

**Discussion by michael-s-molina** [apache/superset#20316] [General]
Hello Superset Community,

This discussion is to get feedback on the next `1.5.2rc1` patch release.

Please submit any cherries by **June 30th**. After that period, the release will be considered closed and new cherries should go to the next patch release. 

Since `1.5.2` will be a patch release, no new features are allowed. New features will be included in the next Superset major release (`2.0.0`) which will arrive pretty soon. If you have any cherries you'd like to include, just add a comment to the thread.

Thanks,
The Apache Superset Team

**Comment by iercan** [comment]
We'd appreciate if https://github.com/apache/superset/pull/20058 included. Thanks

**Comment by kamalkeshavani-aiinside** [comment]
Please add below:
https://github.com/apache/superset/pull/20235
https://github.com/apache/superset/pull/20178
https://github.com/apache/superset/pull/20151

**Reply by kamalkeshavani-aiinside** [reply]
Also if we can add a small fix for this:
https://github.com/apache/superset/issues/20517

**Comment by mdeshmu** [comment]
Please include these.
[20550](https://github.com/apache/superset/pull/20550)
[1056](https://github.com/apache-superset/superset-ui/pull/1056) 
[20178](https://github.com/apache/superset/pull/20178)
[20173](https://github.com/apache/superset/pull/20173)
[20348](https://github.com/apache/superset/pull/20348)

**Comment by miqbalrp** [comment]
Hi, appreciate if we can include this:
[19506 ](https://github.com/apache/superset/issues/19506)

Thank you.

**Reply by sujithzebpay** [reply]
Hi,
Is it possible for us to include the PR https://github.com/apache/superset/pull/14688 in any of the upcoming release.

**Reply by michael-s-molina** [reply]
Hi @sujithzebpay, https://github.com/apache/superset/pull/14688 is still a draft and there are many discussions happening right now that will impact the POC solution. You may expect drilling features a little further this year.

**Comment by sandeep-devarapalli** [comment]
Hi Team, where can I check out the expected features in the 2.0.0 release?

**Reply by michael-s-molina** [reply]
Hi @sandeep-devarapalli. You'll be able to see them on the 2.0 rc1 discussion and release notes as soon as we assemble the release. If you want a sneak peek, here's the [2.0 branch](https://github.com/apache/superset/tree/2.0) where you can check the commits.

**Comment by Thelin90** [comment]
Please include this fix that was done over the weekend (`not being able to query trino tables after version version 1.5.1`):

Issue Bug: https://github.com/apache/superset/issues/20428
PR (merged): https://github.com/apache/superset/pull/20434

**Comment by sujiplr** [comment]
@michael-s-molina  This issue is still there ? Any idea when we will be going to integrate this? Cache warmup will be a great feature for the data layers having more number of datasets/slices and can can help in huge performance improvements.

https://github.com/apache/superset/pull/15713

**Reply by michael-s-molina** [reply]
Hi @sujiplr. We need someone from the community to continue the work. Right now, the draft PR has some pending questions and it's closed due to inactivity.

**Comment by MarcinZegar** [comment]
Any timeline for 1.5.2?

**Reply by michael-s-molina** [reply]
Hi @MarcinZegar. We're currently voting on Superset 2.0 which includes all these fixes and many more. As soon as 2.0 is released, we'll work on 1.5.2.

**Reply by kamalkeshavani-aiinside** [reply]
@michael-s-molina Any updates?

**Reply by michael-s-molina** [reply]
@kamalkeshavani-aiinside 1.5.2rc1 was submitted for a vote!

**Comment by dondelicaat** [comment]
This would be great https://github.com/apache/superset/issues/20565

**Comment by michael-s-molina** [comment]
We just submitted 1.5.2rc1 for a vote!

The release candidate:
https://dist.apache.org/repos/dist/dev/superset/1.5.2rc1/

Git tag for the release:
https://github.com/apache/superset/tree/1.5.2rc1

The Change Log for the release:
https://github.com/apache/superset/blob/1.5.2rc1/CHANGELOG.md

The Updating instructions for the release:
https://github.com/apache/superset/blob/1.5.2rc1/UPDATING.md

Public keys are available at:
https://www.apache.org/dist/superset/KEYS

**Comment by michael-s-molina** [comment]
1.5.2rc1 did NOT pass due to some fixes needed: https://lists.apache.org/thread/foxo7bkgzrrjo40yqwcd694rzm11x2hf

1.5.2rc2 is up for voting: https://lists.apache.org/thread/9cvn1o9v5jxtmovg7l8jzd41jr80873n

The release candidate:
https://dist.apache.org/repos/dist/dev/superset/1.5.2rc2/

Git tag for the release:
https://github.com/apache/superset/tree/1.5.2rc2

The Change Log for the release:
https://github.com/apache/superset/blob/1.5.2rc2/CHANGELOG.md

The Updating instructions for the release:
https://github.com/apache/superset/blob/1.5.2rc2/UPDATING.md

Public keys are available at:
https://www.apache.org/dist/superset/KEYS

**Comment by michael-s-molina** [comment]
1.5.2rc2 did NOT pass due to some fixes needed: https://lists.apache.org/thread/9cvn1o9v5jxtmovg7l8jzd41jr80873n

1.5.2rc3 is up for voting: https://lists.apache.org/thread/f1r601j81dpsj547c97b20qtjlp2j06t

The release candidate:
https://dist.apache.org/repos/dist/dev/superset/1.5.2rc3/

Git tag for the release:
https://github.com/apache/superset/tree/1.5.2rc3

The Change Log for the release:
https://github.com/apache/superset/blob/1.5.2rc3/CHANGELOG.md

The Updating instructions for the release:
https://github.com/apache/superset/blob/1.5.2rc3/UPDATING.md

Public keys are available at:
https://www.apache.org/dist/superset/KEYS
